app/models/concerns/control_objective_items/finding_pdf_data.rb

```diff
@@ -39,51 +50,52 @@ def get_initial_finding_attributes finding, hide, show
       end
 
       if finding.origination_date.present? && hide.exclude?('origination_date')
-        body << "<b>#{finding.class.human_attribute_name('origination_date')}:"+
-          "</b> #{finding_origination_date_text_for finding}\n"
+        body << "<b>#{custom_labels[:origination_date] || finding.class.human_attribute_name('origination_date')}: " +
+        "</b> #{finding_origination_date_text_for finding}\n"
       end
 
-      body << finding_repeated_text_for(finding, show)
+      body << finding_repeated_text_for(finding, show) if hide.exclude?('repeated')
+
+      body
     end
 
-    def get_weakness_attributes finding, hide
+    def get_weakness_attributes finding, hide, custom_labels
       body = ''
 
       if finding.risk_text.present?
-        body << "<b>#{Weakness.human_attribute_name('risk')}:</b> " +
+        body << "<b>#{custom_labels[:risk] || Weakness.human_attribute_name('risk')}:</b> " +
           "#{finding.risk_text.chomp}\n"
       end
 
       if !HIDE_WEAKNESS_EFFECT && finding.effect.present?
-        body << "<b>#{Weakness.human_attribute_name('effect')}:</b> " +
+        body << "<b>#{custom_labels[:effect] || Weakness.human_attribute_name('effect')}:</b> " +
           "#{finding.effect.chomp}\n"
       end
 
       if finding.audit_recommendations.present? && hide.exclude?('audit_recommendations')
-        body << "<b>#{Weakness.human_attribute_name('audit_recommendations')}: " +
+        body << "<b>#{custom_labels[:audit_recommendations] || Weakness.human_attribute_name('audit_recommendations')}: " +
           "</b>#{finding.audit_recommendations}\n"
       end
 
       body
     end
 
-    def get_finding_answer finding
+    def get_finding_answer finding, custom_labels
       body = ''
 
       if finding.answer.present?
-        body << "<b>#{finding.class.human_attribute_name('answer')}:</b> " +
-          "#{finding.answer.chomp}\n"
+        body << "<b>#{custom_labels[:answer] || finding.class.human_attribute_name('answer')}:</b> " +
+        "#{finding.answer.chomp}\n"
```

Comment:
Este si va con los espacios.
```suggestion
          "#{finding.answer.chomp}\n"
```